src/server/qgsmapserverutils.cpp

```diff
@@ -1,84 +0,0 @@
-/***************************************************************************
-    qgsmapserverutils.cpp
-    ---------------------
-    begin                : August 2010
-    copyright            : (C) 2010 by Marco Hugentobler
-    email                : marco dot hugentobler at sourcepole dot ch
- ***************************************************************************
- *                                                                         *
- *   This program is free software; you can redistribute it and/or modify  *
- *   it under the terms of the GNU General Public License as published by  *
- *   the Free Software Foundation; either version 2 of the License, or     *
- *   (at your option) any later version.                                   *
- *                                                                         *
- ***************************************************************************/
-#include "qgsmapserverutils.h"
-#include "qgslogger.h"
-#include <stdlib.h>
-#include <time.h>
-#include <QDir>
-#include <QFileInfo>
-#include <QTextStream>
-
-QString QgsMapServerUtils::createTempFilePath()
```

Comment:
I'm very glad to see these methods go - Coverity has been flagging them as unsafe for as long as we've been running scan over QGIS!